src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/EmbedBlockViewModels.cs

```diff
@@ -0,0 +1,27 @@
+#nullable enable
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels;
```

Comment:
This file should go in the `Admin.Requests` namespace under `EmbedBlockRequests` (not `EmbedBlockViewModels`)